### PR TITLE
[DO-588] Add info on Payment GW specifics

### PR DESCRIPTION
**Pull request by Hdd87**

### CODE REVIEW

File: content/2.application.md [hunk 1]
```diff
@@ -42,10 +42,13 @@ The Participant node is a node in the Corda business network with the Payments C
 ### Payments CorDapp
 
 The Payments CorDapp is a custom CorDapp developed by the Participant or the BNO using the Gevamu Payments SDK. 
+In order to connect the CorDapp with the Gevamu gateway associated with the PSP, the developer must include details of the PSP.
```
Comment (d0rich) [inline content/2.application.md:45]: Developer shouldn't include PSP information. Maybe you are trying to say that every payment initiation request requires specific gateway to be specified.

**Review by d0rich — CHANGES_REQUESTED** [review]

File: content/2.application.md (hunk 1 above)
Comment (lingvolab) [reply content/2.application.md:45]: I believe that the requested change is not related to this section at all. It's about Gevamu Payment Gateway, not the CorDapp

File: content/2.application.md [hunk 2]
```diff
@@ -42,10 +42,13 @@ The Participant node is a node in the Corda business network with the Payments C
...
+Therefore, with minimal changes to the CorDapp, it can work with different PSPs.
 
 ### Gateway node
 
 The Gateway node is a node in the Corda business network where the Gevamu Payments Gateway is installed as a CorDapp. The Gateway node communicates on-chain with the Participant’s CorDapp based on the Gevamu Payments SDK via a proprietary API; the connection with the PSP is off-chain.  
+Each Gevamu gateway is custom written to work with a specific PSP.
```
Comment (lingvolab) [inline content/2.application.md:51]:
```suggestion
Each Gevamu Payments Gateway is custom-written to work with a specific PSP.
```

Comment (lingvolab) [reply content/2.application.md:51]: @Hdd87 I'd move this sentence closer to the beginning of the paragraph - let's put it after the first sentence.

File: content/2.application.md (hunk 1 above)
Comment (lingvolab) [reply content/2.application.md:45]: @Hdd87 @d0rich Let's remove it from this section

**Review by lingvolab — CHANGES_REQUESTED** [review]

**Review by d0rich — APPROVED** [review]

**Review by lingvolab — COMMENTED** [review]
All is good from the language perspective, I'd like @ASitnikov to take a look at the logical side of the edit

**Review by lingvolab — APPROVED** [review]

**Pull request merged**